src/catalog.h

```diff
@@ -1176,6 +1177,58 @@ enum Anum_remote_data_node_name_idx
 	_Anum_remote_txn_data_node_name_idx_max,
 };
 
+/********************************************
+ *
+ * table to track chunk copy/move operations
+ *
+ ********************************************/
+
+#define CHUNK_COPY_ACTIVITY_TABLE_NAME "chunk_copy_activity"
+
+enum Anum_chunk_copy_activity
+{
+	Anum_chunk_copy_activity_id = 1,
+	Anum_chunk_copy_activity_operation_id,
+	Anum_chunk_copy_activity_backend_pid,
+	Anum_chunk_copy_activity_completed_stage,
+	Anum_chunk_copy_activity_time_start,
+	Anum_chunk_copy_activity_chunk_id,
+	Anum_chunk_copy_activity_source_node_name,
+	Anum_chunk_copy_activity_dest_node_name,
+	Anum_chunk_copy_activity_delete_on_src_node,
+	_Anum_chunk_copy_activity_max,
+};
+
+#define Natts_chunk_copy_activity (_Anum_chunk_copy_activity_max - 1)
+
+typedef struct FormData_chunk_copy_activity
```

Comment:
So, unless I am mistaken, this Form isn't actually used anywhere in the code, which is a bit odd. Shouldn't it actually be embedded in the `ChunkCopyData` structure? It seems that `ChunkCopyData` is replicating a lot of fields in this struct so that there is no use for the Form.

Reply:
This happened probably because Dmitry developed `ChunkCopyData` structure and then I added the catalog table later. Let's address this as part of the restructuring issue?